packages/inventory-compliance/src/Constants/Constants.js

```diff
@@ -12,7 +12,9 @@ export const SEVERITY_COLUMN = 2;
+const LowSeverityIcon = <svg width="1em" height="1em" viewBox="0 0 18 18" role="img" style={{ verticalAlign: '-0.125em' }} xmlns="http://www.w3.org/2000/svg"><path d="M2 0h14a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H2a2 2 0 0 1-2-2V2a2 2 0 0 1 2-2zm6.67 10.46a1.67 1.67 0 1 0 0 3.338 1.67 1.67 0 0 0 0-3.338zm-1.586-6l.27 4.935a.435.435 0 0 0 .434.411H9.55c.232 0 .422-.18.435-.411l.27-4.936A.435.435 0 0 0 9.818 4h-2.3c-.25 0-.448.21-.435.46z" fill="#3A9CA6" fillRule="evenodd"/></svg>;
+
+export const EmptyStateImage = props => <svg {...props} style={{ fill: 'var(--pf-global--Color--200)' }} role="img" width="3em" version="1.1" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" viewBox="0 0 138.8 95"><g><g><path d="M115.3,24.1C110.9,9.9,97.6,0,82.7,0c-15,0-28.2,9.8-32.6,24c-2.4-0.9-5-1.3-7.6-1.3 c-10.6,0-19.7,7.3-22.1,17.5c-9.5,0.3-17.5,6.9-19.6,16.2c-0.1,0.4,0,0.9,0.3,1.2c0.3,0.3,0.7,0.5,1.1,0.5h48.7 c-0.2,1.5-0.3,3.1-0.3,4.6C50.6,80.5,65.1,95,82.8,95c17.8,0,32.2-14.5,32.2-32.2c0-1.6-0.1-3.1-0.3-4.6H136 c0.7,0,1.3-0.5,1.4-1.2c0.3-1.7,0.4-3.3,0.4-4.9C137.8,38.6,128.4,26.9,115.3,24.1z M82.8,92c-16.1,0-29.3-13.1-29.3-29.3 c0-16.1,13.1-29.3,29.3-29.3c16.1,0,29.3,13.1,29.3,29.3C112.1,78.9,99,92,82.8,92z M134.8,55.3h-20.6 C110.8,41.1,98,30.5,82.8,30.5c-15.2,0-28,10.6-31.3,24.8H4.1C6.5,48,13.3,43,21.3,43h0l0.2,0c0.7,0.1,1.3-0.5,1.4-1.2 c1.7-9.5,10-16.4,19.6-16.4c2.6,0,5.1,0.5,7.5,1.5c1.5,0.7,4.6,2.5,6.6,5.8c0.2,0.4,0.7,0.6,1.1,0.5c0.1,0,0.2-0.1,0.4-0.1 c0.5-0.3,0.7-0.9,0.4-1.5c-1.7-2.8-4.1-4.7-5.9-5.8c3.7-13.6,16-23.1,30.2-23.1c13.6,0,25.6,8.7,29.7,21.5c-3.6,0-6.4,1.3-6.5,1.4 c-0.5,0.2-0.8,0.8-0.6,1.4c0.2,0.5,0.8,0.8,1.4,0.6c0,0,3.8-1.5,8-0.8c11.8,2.6,20.4,13,20.4,25.2C135,53.1,134.9,54.2,134.8,55.3 z"/></g><g><path d="M97.3,48.4c0,0,0-0.1,0-0.1c0.1-0.3-0.1-0.6-0.4-0.7l-4.4-2.1c-0.1,0-0.2-0.1-0.3-0.1h-18 c-0.1,0-0.2,0-0.3,0.1l-5,2.1c-0.3,0.1-0.5,0.4-0.4,0.8c0,0,0,0.1,0,0.1v15.8c0,0,0,0,0,0l0,15.8c0,0.4,0.3,0.7,0.7,0.7h27.5 c0.4,0,0.7-0.3,0.7-0.7V64.3c0,0,0,0,0,0L97.3,48.4z M93.7,75.6h2.2v1h-2.2c-0.3,0-0.5-0.2-0.5-0.5C93.2,75.8,93.4,75.6,93.7,75.6 z M92.3,76.1c0,0.8,0.6,1.4,1.4,1.4h2.2v1.8H69.8v-6.5h26.1v1.8h-2.2C92.9,74.7,92.3,75.3,92.3,76.1z M93.7,67.9h2.2v1h-2.2 c-0.3,0-0.5-0.2-0.5-0.5C93.2,68.1,93.4,67.9,93.7,67.9z M92.3,68.4c0,0.8,0.6,1.4,1.4,1.4h2.2v1.7H69.8V65h26.1v2h-2.2 C92.9,67,92.3,67.6,92.3,68.4z M93.7,59.8h2.2v1h-2.2c-0.3,0-0.5-0.2-0.5-0.5C93.2,60,93.4,59.8,93.7,59.8z M92.3,60.3 c0,0.8,0.6,1.4,1.4,1.4h2.2v1.8H69.8V57h26.1v1.8h-2.2C92.9,58.8,92.3,59.5,92.3,60.3z M72,47.7l2.3-1H92l2,1H72z M93.7,52h2.2v1 h-2.2c-0.3,0-0.5-0.2-0.5-0.5C93.2,52.3,93.4,52,93.7,52z M92.3,52.5c0,0.8,0.6,1.4,1.4,1.4h2.2v1.7H69.8v-6.5h26.1v2h-2.2 C92.9,51.1,92.3,51.7,92.3,52.5z"/></g></g></svg>;
```

Comment:
Please don't use custom SVGs, PF should have all icons. You can use `CloudServerIcon` for this one